random_walk.cpp

```diff
@@ -5,12 +5,14 @@
 #include <fstream>
 #include <iomanip>
 #include <random>
+using namespace std;
```

Comment:
Are we sure we want to declare std? I don't know the convention on this, I was avoiding it to make everything explicit. If we were to use a different namespace in the future, we could have conflicting functions.

Reply:
I looked on stack overflow, there seems to be a pretty strong consensus to not declare this. 

https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice